src/main/resources/templates/add_product.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Seems that you have added an irrelevant file to the project. Remove this file if not needed.